unit1-intro-to-programming-in-javascript/README.md

```diff
@@ -61,6 +62,11 @@
   - Creating new DOM elements with `.append()` and `.prepend()`
   - Reading and validating form content with `.val()`
   - Interacting with sets of jQuery DOM elements with `.each()`
+  - Browser Events
+    - Event handlers and the concept of callbacks (+ brief intro to asynchronous nature of javascript)
+    - Event propagation: bubbling vs capture.
+    - jQuery event handling methods: `.click()` vs `.on()`
+    - Using event propagation to efficiently bind to dynamic elements ie: `.on('click', '.child-selector', cb)`
```

Comment:
👍 i totally missed this, and this is required for many of the learning outcomes. I have "anonymous inline function definitions" in week 2, but i think revisiting it in the context of events is the right thing to do.